modules/sendgrid/README.md

```diff
@@ -0,0 +1,35 @@
+# Sendgrid
+
+Authenticates your domain with sendgrid, assumes Sendgrid API key is available in secret-manager
+
+## Notes
+This implementation relies on a bash-script to fetch the CNAMEs required for sendgrid domain authentication, the bashscript must be implemented in an idempotent way where it always returns the same result for terraform state.
+
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+## Requirements
+- jq
+- curl	No requirements.
+- Api key to be stored in AWS secret manager
+- HostedZone to be properly configured as root of domain in Route53
```

Comment:
should these be moved to `## Notes` instead of requirements since theyre generated

Reply:
Yep!

Reply:
Otherwise next time someone runs the pre-commit hook that will be wiped out.